x-pack/plugins/index_lifecycle_management/__jest__/policy_table.test.tsx

```diff
@@ -197,7 +199,7 @@ describe('policy table', () => {
     const numberOfIndices = testPolicy.indices.length;
     const numberOfIndexTemplates = testPolicy.indexTemplates.length;
     expect(firstRow).toBe(
-      `testy0${numberOfIndices}${numberOfIndexTemplates}${testPolicy.version}${testDateFormatted}Actions`
+      `Nametesty0Linked index templates${numberOfIndexTemplates}Linked indices${numberOfIndices}Modified date${testDateFormatted}`
```

Comment:
I find this a little hard to parse as well 😄 . Is there a better way to test the values?